src/ConnectorRuntimeConfig.ts

```diff
@@ -20,7 +20,9 @@ export interface ConnectorRuntimeConfig extends RuntimeConfig {
 
     logging: log4js.Configuration;
 
-    modules: Record<string, ConnectorRuntimeModuleConfiguration>;
+    modules: Record<string, ConnectorRuntimeModuleConfiguration> & {
+        decider: DeciderModuleConfiguration;
```

Comment:
The `DeciderModule` is for Requests and their Responses only, right? It may makes sense to rename the module to `requestDecider` in the configuration then, because it is more self-explanatory.

Reply:
Okay for me 👍